packages/openapi3/src/schema-emitter.ts

```diff
@@ -813,6 +822,24 @@ export class OpenAPI3SchemaEmitter extends TypeEmitter<
       "deprecated",
     );
 
+    if (this.#xmlModule) {
```

Comment:
xml library should be manually import by tsp author, right?
If so, maybe throw an error or add a warning log (see what others do?) when xml's decorator is used but xml library is not import?